bedrock/base/templates/includes/protocol/navigation/navigation.html

```diff
@@ -4,7 +4,7 @@
  file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
-{% if switch('m24-website-refresh', ['en']) %}
+{% if switch('m24-website-refresh') and ftl_file_is_active('navigation_refresh') %}
```

Comment:
Issue: (blocking) Same problem as the footer - we should check for both nav and footer here as well, since the CSS files won't be what we need if only the nav is translated.